uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/sickness-leave-request-instance_test.js

```diff
@@ -0,0 +1,163 @@
+define([
+  'mocks/helpers/helper',
+  'mocks/apis/leave-request-api-mock',
+  'mocks/apis/option-group-api-mock',
+  'leave-absences/shared/models/instances/sickness-leave-request-instance',
+], function (helper) {
+  'use strict';
+
+  describe('SicknessRequestInstance', function () {
+    var expectedError, instance, LeaveRequestAPI, $provide, promise, $q, requestData, $rootScope;
+
+    beforeEach(module('leave-absences.models.instances', 'leave-absences.mocks', function (_$provide_) {
+      $provide = _$provide_;
+    }));
+
+    beforeEach(inject(function (_LeaveRequestAPIMock_, _OptionGroupAPIMock_) {
+      $provide.value('LeaveRequestAPI', _LeaveRequestAPIMock_);
+      $provide.value('OptionGroup', _OptionGroupAPIMock_);
+    }));
+
+    beforeEach(inject([
+      'SicknessRequestInstance', '$rootScope', 'LeaveRequestAPI', '$q',
+      function (_SicknessRequestInstance_, _$rootScope_, _LeaveRequestAPI_, _$q_) {
```

Comment:
Can we have the global injections first, then rest in alphabetical order,
`_$q_, _$rootScope_,  _LeaveRequestAPI_, _SicknessRequestInstance_`